File: Java/Tarjeta.java
package Java;

import java.sql.Date;

public class Tarjeta extends Pay{

    String numero;
    String cvv;
    Date vencimiento;
    
    public Tarjeta(Integer id, String numero, String cvv, Date vencimiento) {
        super(id);
        this.numero = numero;
        this.cvv = cvv;
        this.vencimiento = vencimiento;
    }

    


    
}

File: PHP/Tarjeta.php
<?php

require_once('pay.php');

class Tarjeta extends Pay{

    public $numero;
    public $cvv;
    public $vencimiento;
    
    public function __construct($id, $numero, $cvv, $vencimiento)
    {
        parent:: __construct($id, $numero, $cvv, $vencimiento);

        $this -> numero = $numero;
        $this -> cvv = $cvv;
        $this -> vencimiento = $vencimiento;
        
    }
        
}
